packages/logrus/challenge-3-advanced-configuration-and-hooks/run_test.sh

```
@@ -0,0 +1,71 @@
+mv "$TEMP_DIR/solution.go" "$TEMP_DIR/solution-template.go"
+
+echo "Running tests for user '$USERNAME'..."
+
+# Navigate to the temporary directory
+pushd "$TEMP_DIR" > /dev/null
+
+# Tidy up dependencies to ensure everything is consistent
+echo "Tidying dependencies..."
+go mod tidy || {
+    echo "Failed to tidy dependencies."
+    popd > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)